EXILED/Exiled.Events/Handlers/Player.cs

```diff
@@ -70,12 +70,16 @@ public class Player
         public static Event<EarningAchievementEventArgs> EarningAchievement { get; set; } = new();
 
         /// <summary>
-        /// Invoked before using an <see cref="API.Features.Items.Item"/>.
+        /// Invoked before the player starts to use an <see cref="API.Features.Items.Item"/>. In other words, it is invoked just before the animation starts.
         /// </summary>
+        /// <remarks>
+        /// Will be invoked even if the <see cref="API.Features.Items.Item"/> is on cooldown.
+        /// Items that will invoke this event: SCP-1853, SCP-207, Anti-Cola, all medical, SCP-244, SCP-1576 and SCP-268. Candies are not included.
```

Comment:
This should just say "Candies don't trigger this event" since all the other ones are just all the `Usable` items.

Reply:
Legit just thought this myself